### DISCUSSION TITLE
Can any github stuff help me solve this?

### DISCUSSION THREAD

**Discussion by somebody-someone** [community/community#42505] [GitHub Education]
The new year is coming. Can I request the student pack as my new year gift? All my profiles and proofs are ready, and please just let my pass that annoying robot check ok? I'm realy a student, which is even not possible to let the github know.

**Comment by somebody-someone** [comment]
If there is something I need to do, please tell me directly, instead of keeping reminding me the things I'd already done.

**Reply by somebody-someone** [reply]
Anyway after a long time, now I'm able to submit my application and approved. Nice result.

**Comment by jfullstackdev** [comment]
there are a lot of similar posts, not all will be successful though,

but maybe a Staff member here will see this post that might help you getting through this issue
